src/lib/philips.ts

```diff
@@ -94,7 +94,44 @@ const philipsModernExtend = {
             result.exposes.push(...exposeEndpoints(e.enum("effect", ea.SET, effects), args.endpointNames));
         }
 
-        const customCluster = m.deviceAddCustomCluster("manuSpecificPhilips3", {
+        const customClusterFC00 = m.deviceAddCustomCluster("manuSpecificPhilips", {
+            ID: 0xfc00,
+            manufacturerCode: Zcl.ManufacturerCode.SIGNIFY_NETHERLANDS_B_V,
+            attributes: {
+                config: {ID: 49, type: Zcl.DataType.BITMAP16},
+            },
+            commands: {},
+            commandsResponse: {
+                hueNotification: {
+                    ID: 0,
+                    parameters: [
+                        {name: "button", type: Zcl.DataType.UINT8},
+                        {name: "unknown1", type: Zcl.DataType.UINT24},
+                        {name: "type", type: Zcl.DataType.UINT8},
+                        {name: "unknown2", type: Zcl.DataType.UINT8},
+                        {name: "time", type: Zcl.DataType.UINT8},
+                        {name: "unknown2", type: Zcl.DataType.UINT8},
+                    ],
+                },
+            },
+        });
+
+        const customClusterFC03 = m.deviceAddCustomCluster("manuSpecificPhilips2", {
+            ID: 0xfc03,
+            manufacturerCode: Zcl.ManufacturerCode.SIGNIFY_NETHERLANDS_B_V,
+            attributes: {
+                state: {ID: 0x0002, type: Zcl.DataType.OCTET_STR},
+            },
+            commands: {
+                multiColor: {
+                    ID: 0,
+                    parameters: [{name: "data", type: Zcl.BuffaloZclDataType.BUFFER}],
+                },
+            },
+            commandsResponse: {},
+        });
+
+        const customClusterFC01 = m.deviceAddCustomCluster("manuSpecificPhilips3", {
```

Comment:
Any chance we can have `const customClusterFC03` appear after `const customClusterFC01`? I realize this means `manuSpecificPhilips3` will come before `manuSpecificPhilips2`, but now that we're getting a better and better idea what these do, hopefully we can (at some later point, in a future PR) rename the `manuSpecific...` to more meaningful names.

Reply:
Done!